feature-libs/organization/user-registration/components/verification-token-form/verification-token-form.component.ts

```diff
@@ -93,11 +93,12 @@ export class RegisterVerificationTokenFormComponent implements OnInit {
           {}
         );
         this.routingService.go(
-          this.featureConfigService.isEnabled('authorizationCodeFlowByDefault')
+          this.featureConfigService.isEnabled(
+            'nativeSupportForRedirectingOAuthFlows'
+          )
```

Comment:
We could probably create another feature flag to fix string-based router destinations, but if we wanted to piggy-back on another flag, then it should be part of the changes to enhance support for oAuth flows involving a redirect.